packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -348,11 +348,14 @@ class NavigatorObserver {
   /// The [Navigator] replaced `oldRoute` with `newRoute`.
   void didReplace({ Route<dynamic> newRoute, Route<dynamic> oldRoute }) { }
 
-  /// The [Navigator]'s routes are being moved by a user gesture.
+  /// The [Navigator]'s route `route` is being moved by a user gesture.
```

Comment:
We should explain who needs to call this and when.  From the sound of it, it's just a notification.

The doc should link to and summarize the role of didStopUserGesture as well as Hero.transitionFromUserGestures.

Reply:
Cross referenced didStopUserGesture. 

Though this is a more generic callback API than heroes. We should probably not reference users of the API from the API.